src/assets/wise5/common/file/file.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { isAudio, isImage, isVideo } from './file';
+
+describe('isImage()', () => {
+  it('should return true if string ends in an image extension type', () => {
+    const filename = 'image.jpg';
+    expect(isImage(filename)).toBeTruthy();
```

Comment:
I think it's fine to pass in the string directly to the function. 
```suggestion
    expect(isImage('image.jpg')).toBeTruthy();
```